fluss-client/src/main/java/com/alibaba/fluss/client/metadata/MetadataUpdater.java

```diff
@@ -300,7 +300,10 @@ private static Cluster initializeCluster(Configuration conf, RpcClient rpcClient
             RpcClient rpcClient, InetSocketAddress address) {
         ServerNode serverNode =
                 new ServerNode(
-                        -1, address.getHostName(), address.getPort(), ServerType.TABLET_SERVER);
+                        -1,
+                        address.getAddress().getHostAddress(),
```

Comment:
use `address.getHostString()` instead? which I think is safer.

Reply:
Agreed, `getHostString` is better, I've fixed the conflicts, please take a look again.